api/src/modules/h3-data/h3-data.repository.ts

```diff
@@ -34,16 +38,9 @@ export class H3DataRepository extends Repository<H3Data> {
    * @param resolution: An interger between 1 (min resolution) and 6 (max resolution).
    * Resolution validation done at route handler
    *
-   * @returns: Single object with the h3Index as property key, and its value
-   *
-   * @debt: This should be done on a single raw sql query, but I couldn't make it work
-   * Super Senior powers requested
    */
 
-  async getH3ByIdAndResolution(
-    h3Id: string,
-    resolution: number,
-  ): Promise<H3IndexValueData> {
+  async getH3ByIdAndResolution(h3Id: string, resolution: number): Promise<any> {
```

Comment:
Can the return type be improved?

Reply:
Yes of course, good catch!

Fix -> 0752f2ebaedcc5d72f18086f93ed6623ebc71e32